generate-spec

```diff
@@ -235,9 +235,13 @@ if (count($parsedRoutes) == 0) {
 
 $routes = [];
 foreach ($parsedRoutes as $key => $value) {
-	if ($key != "routes" && $key != "ocs") {
+	$isOCS = $key === "ocs";
```

Comment:
What about https://github.com/nextcloud/server/blob/f5c361cf44739058b79f322576a1bad2d8c142d9/lib/private/AppFramework/Routing/RouteConfig.php#L171-L187 btw? :)

Reply:
I have a WIP branch from months ago, but since the resources are not really used anywhere I didn't mind continuing it. Will open an issue to track it.